app/uk/gov/hmrc/merchandiseinbaggagefrontend/model/declaration/Declaration.scala

```diff
@@ -59,7 +62,29 @@ object PriceOfGoods {
 case class GoodsEntry(typeOfGoods: String,
                       maybeCountryOfPurchase: Option[String] = None,
                       maybePriceOfGoods: Option[PriceOfGoods] = None,
-                      maybeTaxDue: Option[CurrencyAmount] = None)
+                      maybeTaxDue: Option[CurrencyAmount] = None) {
+  def toSummaryList(implicit messages: Messages) = {
```

Comment:
This method belongs on and is easier to code on the Goods case class rather than GoodsEntry because the optionality is removed. The method should only be used with complete goods entries (where we can construct a Goods instance safely).